### DISCUSSION TITLE
r/anji-tianchi-forest-camping

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#19] [Announcements]
# r/anji-tianchi-forest-camping

游记 | 在安吉大山里发现神仙级森林露营地

https://blog.ops-coffee.cn/r/anji-tianchi-forest-camping

**Comment by algery** [comment]
金钱松森林公园可以等秋天再来，又是别样风景

**Comment by mlsnatalie** [comment]
去年秋天去了，满山黄叶，特别漂亮。公园已经建好，不能开车进去了。门口有停车场。

**Reply by ops-coffee** [reply]
感谢你，更新信息
